patches/chrome-installer-mac-signing-pipeline.py.patch

```diff
@@ -1,12 +1,34 @@
 diff --git a/chrome/installer/mac/signing/pipeline.py b/chrome/installer/mac/signing/pipeline.py
-index 898c0d4542bb3474d654e71a1e4fd5a1355e3897..dad0718097522f2346f690ae7435dbe162cbb964 100644
+index 898c0d4542bb3474d654e71a1e4fd5a1355e3897..a1a32cac6e7e23bda56411b165ff0fe84a620e91 100644
 --- a/chrome/installer/mac/signing/pipeline.py
 +++ b/chrome/installer/mac/signing/pipeline.py
-@@ -221,7 +221,6 @@ def _package_installer_tools(paths, config):
+@@ -192,12 +192,6 @@ def _package_dmg(paths, dist, config):
+         '--volname', config.app_product,
+         '--icon', os.path.join(packaging_dir, icon_file),
+         '--copy', '{}:/'.format(app_path),
+-        '--copy',
+-            '{}/keystone_install.sh:/.keystone_install'.format(packaging_dir),
+-        '--mkdir', '.background',
+-        '--copy',
+-            '{}/chrome_dmg_background.png:/.background/background.png'.format(
+-                packaging_dir),
+         '--copy', '{}/{}:/.DS_Store'.format(packaging_dir, dsstore_file),
+         '--symlink', '/Applications:/ ',
+     ])
+@@ -221,7 +215,6 @@ def _package_installer_tools(paths, config):
          'dirdiffer.sh',
          'dirpatcher.sh',
          'dmgdiffer.sh',
 -        'keystone_install.sh',
          'pkg-dmg',
      )
  
+@@ -338,5 +331,8 @@ def sign_all(orig_paths, config, disable_packaging=False, do_notarization=True):
+                         uuids_to_package_path.keys(), config):
+                     package_path = uuids_to_package_path[result]
+                     notarize.staple(package_path)
++        else:
++            # Copy the notarized app to the orig_paths.output dir where the user expects it.
++            commands.copy_files(dest_dir, orig_paths.output)
```

Comment:
we can copy files in a separate target after the signing

Reply:
In the `brave/build/mac/BUILD.gn` file, I output signed .dmg/.pkg files into the `signed` subdir, then I notarize them from that subdir, and add a directory that outputs the notarized .dmg/.pkg files. This also allows us to avoid a duplicate outputs error in GN because if notarization is not being done, we want the same file names to be created in the `src/out/Release` output directory whether only signing is requested or also if notarization is requested. To work around that I use the `copy_dmg_pkg` target to determine which files to copy to the output directory, depending on whether `notarize` is defined.

Reply:
Can't you just use a different directory to avoid the duplicate outputs? Or better yet, why don't we just plug our own notarize.py into the python path and use the existing calls with `do_notarization` enabled? 
If we did need to keep the patch for some reason we never add comments in patches and there's no reason for the `else` because in this case we know that `do_notarization` is false. When we do have to patch files the patches should be as small as possible.

Reply:
and actually it's not clear to me how this solves the problem anyway because now this target needs to specify the outputs that the copy would have

Reply:
@bridiver can we keep this as-is for now? I believe because we use `--disable-packaging` (to set custom icons, path to install location (`/Applications/` etc). We could follow up on this later (I can create an issue and @mbacchi can make sure it's tracked in the regular Devops meeting). Since this works as-is, this would eat some time ☹️

Reply:
it actually doesn't work as-is. It breaks gn's ability to known when things need to be updated because this script no longer specifies the correct outputs

Reply:
the other target that has the same output won't necessarily run when it should now